### ISSUE TITLE
Save video registration in config file (Trac #76)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#80]
Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/76
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Save video registration in config file", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-15T06:09:18", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120315T06:11:07: ryan@fobel.net commented:

> commit e5b8b63217ac9da988b848ebef8888d9d65a5a79
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Thu Mar 15 02:04:32 2012 -0400
> 
>     Save video registration in config file
>     
>     See #80. Note that only saving is implemented, not loading registration
>     data from the file.

**Comment by cfobel** [comment]
Trac update at 20120315T18:13:07: 
* ryan@fobel.net commented:

> commit 4fb69839680a11572de06102018dc8607cc7ac32
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Thu Mar 15 14:02:54 2012 -0400
> 
>     Modify processing of config file
>     
>     The config file is now processed (and the app options set) by the
>     on_app_init handler of the config_controller class. If other plugins
>     want to be able to access their app options from their on_app_init
>     handler, they should request that the config_controller is scheduled
>     before them.
>     
>     This commit also makes the registration matrix an app option so
>     that it is persistent when the progam is restarted.
>     
>     Fixes #80.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "gui" to "microdrop"
